test/steps/v2_steps.py

```diff
@@ -51,6 +51,36 @@ def mock_response(context, jsonfiles, directory):
     urlopen(req)
 
 
+@given('mock http responses in "{filename}" loaded from "{directory}" with status {status}.')
+def step_impl(context, filename, directory, status):
+    context.expected_status_code = status
```

Comment:
See the comment just above on line 43-49 about how the mocking is done by writing to a flat file. Do you know how feasible it would be to support mocking the status code? Please ping me after you take a look and we can figure out how to proceed.